src/realTimeReporter/utils.ts

```diff
@@ -0,0 +1,11 @@
+import { EVENTS } from '../constants';
+
+export const publishEvent = (event: EVENTS, msg: any): void => {
+  // @ts-ignore
```

Comment:
Will using `@types/node` help?

Reply:
Unfortunately no.
As we use methods from `process` implementation of `Process` interface and haven't such method overload.
![image](https://user-images.githubusercontent.com/31861805/75671763-63f0e880-5c90-11ea-8fdf-d8f9c3d2b23e.png)

Reply:
I found some solution. Please check it out (see the screenshot below).

![image](https://user-images.githubusercontent.com/31861805/75675178-a964e400-5c97-11ea-8b14-5109de9efc46.png)

Reply:
Destructuring will break event emitter (functions will lose their context). Maybe it's not worth wasting too much time trying to make this perfectly typed.

Reply:
Agree. So I just leaved `@ts-ignore` here